<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Polygon3DSymbolWidget</class>
 <widget class="QWidget" name="Polygon3DSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Altitude clamping</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsDoubleSpinBox" name="spinExtrusion">
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelOffset">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cboAltBinding">
     <item>
      <property name="text">
       <string>Vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Centroid</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Rendered facade</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QgsCollapsibleGroupBox" name="groupEdges">
     <property name="title">
      <string>Edges</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="spinEdgeWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsColorButton" name="btnEdgeColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="chkAddBackFaces">
     <property name="text">
      <string>Add back faces</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cboRenderedFacade">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>No Facades</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Walls</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Roofs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Walls and Roofs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Walls, Roofs and Floors</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Culling mode</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="chkInvertNormals">
     <property name="text">
      <string>Invert normals (experimental)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QgsPropertyOverrideButton" name="btnExtrusionDD">
     <property name="text">
      <string>…</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QgsCollapsibleGroupBox" name="groupShading">
     <property name="title">
      <string>Shading</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QgsMaterialWidget" name="widgetMaterial" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QgsPropertyOverrideButton" name="btnHeightDD">
     <property name="text">
      <string>…</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Extrusion</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsDoubleSpinBox" name="spinOffset">
     <property name="minimum">
      <double>-99999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Altitude binding</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cboAltClamping">
     <item>
      <property name="text">
       <string>Absolute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Relative</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Terrain</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cboCullingMode"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMaterialWidget</class>
   <extends>QWidget</extends>
   <header>qgsmaterialwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinOffset</tabstop>
  <tabstop>btnHeightDD</tabstop>
  <tabstop>spinExtrusion</tabstop>
  <tabstop>btnExtrusionDD</tabstop>
  <tabstop>cboAltClamping</tabstop>
  <tabstop>cboAltBinding</tabstop>
  <tabstop>cboCullingMode</tabstop>
  <tabstop>cboRenderedFacade</tabstop>
  <tabstop>chkAddBackFaces</tabstop>
  <tabstop>chkInvertNormals</tabstop>
  <tabstop>groupEdges</tabstop>
  <tabstop>spinEdgeWidth</tabstop>
  <tabstop>btnEdgeColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
